spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /posts' do
+    before(:example) do
+      get '/users/1/posts'
+    end
+
+    it 'returns a 200 status code' do
```

Comment:
Kindly ensure that this test passes.

![image](https://user-images.githubusercontent.com/27709832/204723082-e442baa6-fcbe-460f-af0d-04b934952463.png)